share/completions/pkg.fish

```diff
@@ -165,3 +165,105 @@ complete -c pkg -n '__fish_pkg_is list' -xa '(pkg query "%n")'
 
 # update
 complete -c pkg -n '__fish_pkg_is add update' -s f -l force -d "Force a full download of a repository"
+
+# alias 
+for alias in (pkg alias -q)
+  set -l name (echo $alias | cut -f 1 -d' ')
+  set -l description (echo $alias | cut -f 2- -d' ')
+
+  switch $name
+  case 'all-depends'
+    set description 'Display all dependencies for a given package'
```

Comment:
We use four space indents - you could just run this through `fish_indent -w pkg.fish`